# HG changeset patch
# User Rik <rik@octave.org>
# Date 1430511263 25200
#      Fri May 01 13:14:23 2015 -0700
# Node ID 88c0f8f56a4f4842d906feea40dcf3f20b683293
# Parent  0afc06948ba724c4ab6dfa902397cdc7080c0568
acinclude.m4: Use simpler AC_PREPROC_IFELSE test for checking SETPLACEHOLDERTEXT.

* acinclude.m4 (OCTAVE_CHECK_FUNC_SETPLACEHOLDERTEXT): Replace
AC_COMPILE_IFELSE with AC_PREPROC_IFELSE for SETPLACEHOLDERTEXT function check.

diff --git a/m4/acinclude.m4 b/m4/acinclude.m4
--- a/m4/acinclude.m4
+++ b/m4/acinclude.m4
@@ -412,17 +412,17 @@ dnl FIXME: This test uses a version numb
 dnl        be re-written to actually call the function, but is it worth it?
 dnl
 AC_DEFUN([OCTAVE_CHECK_FUNC_SETPLACEHOLDERTEXT], [
   AC_CACHE_CHECK([whether Qt has SetPlaceholderText function],
     [octave_cv_func_setplaceholdertext],
     [AC_LANG_PUSH(C++)
     ac_octave_save_CPPFLAGS="$CPPFLAGS"
     CPPFLAGS="$QT_CPPFLAGS $CPPFLAGS"
-    AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
+    AC_PREPROC_IFELSE([AC_LANG_PROGRAM([[
         #include <Qt/qglobal.h>
         ]], [[
         #if QT_VERSION < 0x040700
         #error No SetPlacholderText function available.
         #endif
         ]])],
       octave_cv_func_setplaceholdertext=yes,
       octave_cv_func_setplaceholdertext=no)
